flask_app/app.py

```diff
@@ -135,7 +135,7 @@ def get_new_data():
                     logger.info("{}, {}".format(each_resp['latitude'], each_resp['longitude']))
                     new_location = Location(
                         device_id=each_resp['device_id'],
-                        raw=each_resp['raw'],
+                        raw='',
```

Comment:
Here I just converted raw to a string because raw is not present in the new json response. I don't know what raw is used for.